arch/arm/soc/nxp_kinetis/kwx/dts.fixup

```diff
@@ -137,5 +137,7 @@
 #define CONFIG_SPI_1_IRQ_PRI			NXP_KINETIS_DSPI_4002D000_IRQ_0_PRIORITY
 #define CONFIG_SPI_1_CLOCK_NAME			NXP_KINETIS_DSPI_4002D000_CLOCK_CONTROLLER
 #define CONFIG_SPI_1_CLOCK_SUBSYS		NXP_KINETIS_DSPI_4002D000_CLOCK_NAME
+
+#define CONFIG_RTC_0_NAME			NXP_KINETIS_RTC_4003D000_LABEL
```

Comment:
Move this with the rest of the RTC fixups.

Reply:
Change made.